k8s/upgrade-job/src/upgrade/data_plane.rs

```diff
@@ -123,12 +123,14 @@ async fn restart_data_plane(node_name: &str, pod: &Pod, k8s_client: &KubeClientS
             name: pod_name,
             node: node_name.to_string(),
         })?;
+    tracing::info!(node.name = %node_name, "Pod delete command issued");
     Ok(())
 }
 
 /// Wait for the data plane pod to come up on the given node.
-async fn wait_node_drain(rest_client: &RestClientSet) -> Result<()> {
+async fn wait_node_drain(node_name: &str, rest_client: &RestClientSet) -> Result<()> {
     while is_draining(rest_client).await? {
+        tracing::info!(node.name = %node_name, "Waiting for node drain completion");
```

Comment:
```suggestion
        tracing::info!(node.name = %node_name, "Waiting for {} node drain completion", PRODUCT);
```

Because Node and Drain are terms used in the Kubernetes context also...